operatorcert/entrypoints/upload_artifacts.py

```diff
@@ -168,12 +172,19 @@ def upload_results_and_artifacts(args: Any) -> Dict[str, Any]:
     Returns:
         Dict[str, Any]]: Artifacts respones
     """
+    org_id = None
+    if pyxis.is_internal():
+        # External Pyxis gets org_id directly from API key - internally we
+        # have to get it from project
+        project = pyxis.get_project(args.pyxis_url, args.cert_project_id)
```

Comment:
It's not merged yet but in another PR I updated `certification-project-check` to get the project from Pyxis. It'll probably be better to stick to just getting the project once in that step, then adding fields to the output of that step as needed? Please see https://github.com/redhat-openshift-ecosystem/operator-pipelines/pull/22/

Maybe we can just extract the `get_project()` part into a small Python script to be called in the `certification-project-check` step, instead of the curl command I have right now.

Reply:
Just to add to my point above, it looks like this task will also need a field from querying the cert project object: https://issues.redhat.com/browse/ISV-916
So I think the result of getting the cert project object should definitely be available somehow in all the pipeline steps.

Reply:
If you need a step that gets a project details we can create a separate entry point that you can call in a step as CLI and it solves auth part based on if you define cert or API key secrets.